app/src/lang/translations/en-US.yaml

```diff
@@ -2291,3 +2296,145 @@ operations:
     batch: Batch
     batch_size: Batch Size
     cache: Cache
+core_unit_identifier: Core Unit Identifier
```

Comment:
On a second thought: I'd really like to have the units translations nested under a key to make the affiliation clear.
Also I'm not sure whether some of these added translations are really used. For example I cannot see where `core_unit_identifier` is used. Can we clean those up if unused?

Reply:
I just realized that none of the units are needed, at the beginning the unit was a dropdown instead of a text input but now that it is a text input these are no longer needed. They have been removed.

Reply:
Ah okay, I see 👍 Thanks for the clean-up ❤️